cypress/plugins/index.js

```diff
@@ -24,13 +24,25 @@
 const faker = require("faker");
 
 module.exports = (on, config) => {
-	on("task", {
-		stakeholderData() {
-			stakeholder = {
-				displayName: faker.name.findName(),
-				email: faker.internet.email(),
-			};
-			return stakeholder;
-		},
-	});
-};
\ No newline at end of file
+    on("task", {
+        stakeholderData() {
+            stakeholder = {
+                displayName: faker.name.findName(),
+                email: faker.internet.email(),
+            };
+            return stakeholder;
+        },
+    });
+};
+
+module.exports = (on, config) => {
+    on("task", {
+        stakeholdergroupData() {
+            stakeholdergroup = {
+                name: faker.name.findName(),
```

Comment:
Please check for some other option here, as this will generate a full name relevant to person. This may not suit well for the purpose of stakeholder group name !